inc/composer/class-command.php

```diff
@@ -74,8 +76,8 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 			return $this->shell( $input, $output );
 		}
 
-		$output->writeln( '<error>' . $subcommand . ' command not found.</>' );
-		return 1;
+		// Default to start command.
+		return $this->start( $input, $output );
```

Comment:
I think this should check if subcommand is empty, otherwise something like `composer serve foo` will trigger this.

Reply:
Good call, updated